internal/k8s/controller.go

```diff
@@ -969,15 +986,41 @@ func (lbc *LoadBalancerController) syncVirtualServer(task task) {
 		lbc.recorder.Eventf(vsr, vsrEventType, vsrEventTitle, msg)
 
 		if lbc.reportVsVsrStatusEnabled() {
-			virtualServersForVSR := findVirtualServersForVirtualServerRoute(lbc.getVirtualServers(), vsr)
-			err = lbc.statusUpdater.UpdateVirtualServerRouteStatusWithReferencedBy(vsr, state, vsrEventTitle, msg, virtualServersForVSR)
+			vss := []*conf_v1.VirtualServer{vs}
+			err = lbc.statusUpdater.UpdateVirtualServerRouteStatusWithReferencedBy(vsr, state, vsrEventTitle, msg, vss)
 
 			if err != nil {
 				glog.Errorf("Error when updating the status for VirtualServerRoute %v/%v: %v", vsr.Namespace, vsr.Name, err)
 			}
 		}
+
+		vsrkey := fmt.Sprintf("%s/%s", vsr.Namespace, vsr.Name)
+		vsrs = removeVirtualServerRouteByKey(vsrkey, vsrs)
+	}
+
+	reason := "Ignored"
```

Comment:
as was mentioned here https://github.com/nginxinc/kubernetes-ingress/pull/995#discussion_r442570355, this case should not have different reason and message. The VSRS that we report here are no longer referenced by VS (for example, a VS was updated). That's why its better to have the `NoVirtualServersFound` reason and `No VirtualServer references VirtualServerRoute` message